code/game/gamemodes/changeling/powers/escape_restraints.dm

```diff
@@ -0,0 +1,63 @@
+/datum/power/changeling/escape_restraints
+	name = "Escape Restraints"
+	desc = "We evolve more complex joints"
+	helptext = "We can instantly escape from most restraints and bindings, but we cannot do it often."
+	enhancedtext = "More frequent escapes."
+	ability_icon_state = "ling_epinepherine_overdose"
+	genomecost = 2
+	verbpath = /mob/proc/changeling_escape_restraints
+
+//Escape Cuffs
+/mob/proc/changeling_escape_restraints()
+	set category = "Changeling"
+	set name = "Escape Restraints (40)"
+	set desc = "Removes handcuffs and legcuffs instantly."
+
+	var/escape_cooldown = 5 MINUTES		//This is used later to prevent spamming
+	var/mob/living/carbon/human/C = src
+	var/datum/changeling/changeling = changeling_power(40,0,100,CONSCIOUS)
+	if(!changeling)
+		return 0
+	if(world.time < changeling.next_escape)
+		to_chat(src, "<span class='warning'>We are still recovering from our last escape...</span>")
+		return 0
+	if(!(C.handcuffed || C.legcuffed))	// No need to waste chems if there's nothing to break out of
+		to_chat(C, "<span class='warning'>We are are not restrained in a way we can escape...</span>")
+		return 0
+
+	changeling.chem_charges -= 40
+
+	to_chat(C,"<span class='notice'>We slip contort our extremities and slip our cuffs.</span>")
```

Comment:
I believe you have an extra 'slip' in that sentence that should not be there.

Reply:
Good catch, will correct.